### DISCUSSION TITLE
Feature request - Ocean Colour/temperature

### DISCUSSION THREAD

**Discussion by ChrisHarrod** [giswqs/streamlit-geospatial#20] [Ideas]
Firstly, many thanks for a great application!

Further to my request on Twitter, I would like to ask if it is possible to include some oceanographic options such sea surface temperature  or chlorophyll a concentrations. As requested, I looked in the Google Earth Engine catalogue and the products most used by people looking to show variation over time are things like the Sentinal https://developers.google.com/earth-engine/datasets/catalog/COPERNICUS_S3_OLCI or Modis https://developers.google.com/earth-engine/datasets/catalog/NASA_OCEANDATA_MODIS-Aqua_L3SMI

All the best
Chris

**Comment by karelvancamp** [comment]
These can be accessed using geemap.

![image](https://user-images.githubusercontent.com/29794014/194427244-b4f3ba49-fd22-4fe0-85fa-3d941ed248ff.png)
![image](https://user-images.githubusercontent.com/29794014/194427357-3d6aa21d-b43c-49f4-ac5e-d3aa0cab217e.png)
![image](https://user-images.githubusercontent.com/29794014/194427447-76d9a79b-cbaa-47a2-bc74-d17118d404f2.png)

https://www.youtube.com/watch?v=2R51vZSR-y0

**Reply by giswqs** [reply]
Yes, it is doable. I don't have time to work on this at the moment . PR is welcome
